python/OnnxUdf/example.py

```diff
@@ -0,0 +1,35 @@
+import openeo
+
+# This will be required to import onnxruntime in the UDF.
+dependencies_url = "https://artifactory.vgt.vito.be:443/auxdata-public/openeo/onnx_dependencies.zip"
+# You can upload your own model and paste the url here.
+model_url = "https://artifactory.vgt.vito.be:443/auxdata-public/openeo/test_onnx_model.zip"
```

Comment:
The job options expect an archive and will automatically unpack these in separate folders. It could however be an option to provide the link through context (showcasing this functionality as well) and downloading them in the udf through requests. (Is the job archive extracted in a shared drive between the executors? If so moving the fetching inside the udf is less optimal)